ui/components/ui/icon-button/icon-button.stories.tsx

```diff
@@ -34,4 +31,27 @@ type Story = StoryObj<typeof IconButton>;
 
 export const Default: Story = {};
 
-Default.storyName = 'Default';
+export const WithLongLabel: Story = {
+  args: {
+    label: 'This is a very long button label that should be truncated',
+  },
+};
+
+// Test case: Emulate "Bridge button disabled when chain is unsupported"
+export const UnsupportedNetwork: Story = {
+  args: {
+    label: 'Bridge',
+    disabled: true,
+    Icon: <Icon name={IconName.Bridge} color={IconColor.iconDefault} />,
+    tooltipRender: (content) => {
+      // This matches exactly what the test expects
+      const buttonWithAttr = React.cloneElement(content);
+
+      return (
+        <Tooltip title="Unavailable on this network" position="bottom">
+          {buttonWithAttr}
+        </Tooltip>
+      );
+    },
+  },
+};
```

Comment:
Adding 2 more stories to improve storybook coverage and understand the Icon Button behaviour and props

<img width="634" alt="Screenshot 2025-05-16 at 12 35 56 PM" src="https://github.com/user-attachments/assets/1412c3ef-1f12-4696-bc24-81276b65aaad" />

<img width="539" alt="Screenshot 2025-05-16 at 12 36 01 PM" src="https://github.com/user-attachments/assets/33aaa535-77ea-4d56-bdcc-bec1e8455786" />